apps/webClient/src/views/About/index.tsx

```diff
@@ -22,8 +22,8 @@ const About: React.FC<Props> = ({}) => {
 
   return (
     <Grid container>
-      <Hidden only="xs">
-        <Grid item sm={2}>
+      <Hidden only={["xs", "sm", "md"]}>
+        <Grid item md={2}>
```

Comment:
Adjusted these values to the new breakpoints because of this.
![Screenshot from 2020-03-27 16-12-18](https://user-images.githubusercontent.com/16766024/77797854-31949a00-7048-11ea-9100-9a46c14e5a8f.png)

Reply:
Not sure exactly what you mean.  Just pulled the code down and it is not showing the hamburger at md and the content is still no centered.